cadquery/occ_impl/shapes.py

```diff
@@ -1838,6 +1865,59 @@ def paramAt(self: Mixin1DProtocol, d: Union[Real, Vector]) -> float:
 
         return rv
 
+    def params(self: Mixin1DProtocol, pts: Iterable[Vector], tol=1e-6) -> List[float]:
+        """
+        Computes u values closest to given vectors.
+
+        :returns: list of u values.
```

Comment:
Even though it will work fine like this, I think our convention has been to put `return` after the `param` entries. I also don't think it matters, but we have used `return` more times in the code base than `returns`.